GDPR_stuff.txt

```diff
@@ -0,0 +1,28 @@
+The GDPR itself contains 11 chapters and 91 articles. The following are some of the chapters and articles that have the greatest potential impact on security operations:
```

Comment:
It seems like the GDPR branch got merged into the Propublica branch. It might be worth just starting a new branch for this task, and then cherry picking the commits/changes you actually want to keep in this branch. That way, you don't have to fight against the conflicts and revision history.